hydra-core/spec/test_app_templates/Gemfile.extra

```diff
@@ -1,2 +1,3 @@
 gem 'rspec-rails', '~> 3.1', group: :test
 gem 'rspec-its'
+gem 'active-fedora', :github => "projecthydra/active_fedora"
```

Comment:
how about `gem 'active-fedora', '~> 9.0.6'` ? 9.0.8. should be out today or tomorrow.